blocks/location-overview/location-overview.js

```diff
@@ -0,0 +1,94 @@
+import {
+  domEl,
+  div,
+  a,
+  span,
+  h2,
+  p,
+} from '../../scripts/dom-helpers.js';
+
+import {
+  decorateIcons,
+} from '../../scripts/aem.js';
+
+function getLearnMoreIcon() {
+  const learnMoreIcon = domEl('calcite-icon', {
+    icon: 'arrowRight',
+    scale: 's',
+  });
+
+  const iconContainer = div({ class: 'learn-more-icon-container' }, 'Learn more', learnMoreIcon);
+  return iconContainer;
+}
+
+function getPanelIcon(iconPath = '') {
+  const panelIcon = span({
+    class: iconPath || 'location-overview-panel-icon icon icon-2-5d-mesh-48',
+  });
+
+  return panelIcon;
+}
+
+function getContainerContents(hText, pText, panelIconPath) {
+  const panelIcon = getPanelIcon(panelIconPath);
+  const lernMoreIcon = getLearnMoreIcon();
+
+  const overviewCardHeader = h2({
+    class: 'location-overview-card-header',
+  }, hText);
+
+  const overviewCardText = p({
+    class: 'location-overview-card-text',
+  }, pText);
+
+  const contentsWrapper = div({
+    class: 'location-overview-content-wrapper calcite-animate calcite-animate__in-up',
+  }, panelIcon, overviewCardHeader, overviewCardText, lernMoreIcon);
+
+  return contentsWrapper;
+}
+function getLinkWrapper(hText, pText, cardLinkTo) {
+  const linkWrapper = a({
+    class: 'location-overview-link-wrapper',
+    href: cardLinkTo,
+  }, getContainerContents(hText, pText));
+
+  return linkWrapper;
+}
+
+function getLocationOverviewDomElements(cardCount, headingTexts, genericTexts, cardLinks) {
+  // overviewWrapper wraps around all our containers
+  const overviewWrapper = div({ class: 'location-overview-container-wrapper' });
+
+  for (let i = 0; i < cardCount; i += 1) {
+    const card = div({ class: 'location-overview-card-container calcite-mode-dark' }, getLinkWrapper(headingTexts[i], genericTexts[i], cardLinks[i]));
```

Comment:
"calcite-mode-*" should generally either be inherited from the body or defined at the block level (in both cases in the content). An example can be seen in this page: https://main--esri--aemsites.hlx.live/en-us/about/about-esri/americas
![image](https://github.com/user-attachments/assets/93a9d99c-a159-4628-ab56-d65aabb938f0)

Reply:
sounds good. Moving it to the block